src/libs/brand-utils.js

```diff
@@ -1,38 +1,35 @@
-import { brands } from 'src/libs/brands'
+import _ from 'lodash/fp'
+import { brands, defaultBrand } from 'src/libs/brands'
 import { getConfig } from 'src/libs/config'
-import * as Utils from 'src/libs/utils'
 
 
 export const isBrand = brand => {
-  return new RegExp(`^((dev|alpha|perf|staging)\\.)?${brand.hostName}$`).test(window.location.hostname)
+  return new RegExp(`^((dev|alpha|staging)\\.)?${brand.hostName}$`).test(window.location.hostname)
 }
 
-export const isAnvil = () => isBrand(brands.anvil) || getConfig().isAnvil
-export const isBaseline = () => isBrand(brands.baseline) || getConfig().isBaseline
-export const isBioDataCatalyst = () => isBrand(brands.bioDataCatalyst) || getConfig().isBioDataCatalyst
+export const isAnvil = () => getEnabledBrand() === brands.anvil
+export const isBaseline = () => getEnabledBrand() === brands.baseline
+export const isBioDataCatalyst = () => getEnabledBrand() === brands.bioDataCatalyst
 // TODO: Deprecate Datastage (https://broadworkbench.atlassian.net/browse/SATURN-1414)
-export const isDatastage = () => isBrand(brands.datastage) || getConfig().isDatastage
-export const isElwazi = () => isBrand(brands.elwazi) || getConfig().isElwazi
-export const isFirecloud = () => isBrand(brands.firecloud) || getConfig().isFirecloud
-export const isProjectSingular = () => isBrand(brands.projectSingular) || getConfig().isProjectSingular
-export const isRadX = () => isBrand(brands.radX) || getConfig().isRadX
-export const isRareX = () => isBrand(brands.rareX) || getConfig().isRareX
-export const isTerra = () => isBrand(brands.terra) || getConfig().isTerra ||
-  (!isFirecloud() && !isDatastage() && !isAnvil() && !isBioDataCatalyst() && !isBaseline() && !isElwazi() && !isProjectSingular() && !isRadX() && !isRareX())
-
-export const getEnabledBrand = () => Utils.cond(
-  [isAnvil(), () => brands.anvil],
-  [isBaseline(), () => brands.baseline],
-  [isBioDataCatalyst(), () => brands.bioDataCatalyst],
-  [isDatastage(), () => brands.datastage],
-  [isElwazi(), () => brands.elwazi],
-  [isFirecloud(), () => brands.firecloud],
-  [isProjectSingular(), () => brands.projectSingular],
-  [isRadX(), () => brands.radX],
-  [isRareX(), () => brands.rareX],
-  [isTerra(), () => brands.terra],
-  () => brands.terra
-)
+export const isDatastage = () => getEnabledBrand() === brands.datastage
+export const isElwazi = () => getEnabledBrand() === brands.elwazi
+export const isFirecloud = () => getEnabledBrand() === brands.firecloud
+export const isProjectSingular = () => getEnabledBrand() === brands.projectSingular
+export const isRadX = () => getEnabledBrand() === brands.radX
+export const isRareX = () => getEnabledBrand() === brands.rareX
+export const isTerra = () => getEnabledBrand() === brands.terra
+
+export const getEnabledBrand = () => {
+  const forcedBrand = getConfig().brand
+
+  if (!!forcedBrand && _.includes(forcedBrand, _.keys(brands))) {
```

Comment:
This could be a little more concise with [`_.has`](https://lodash.com/docs/4.17.15#has).

```suggestion
  if (!!forcedBrand && _.has(forcedBrand, brands)) {
```

Reply:
Done in [28248a1](https://github.com/DataBiosphere/terra-ui/pull/3625/commits/28248a14eb3561715954ed659cf6fc028a4cc519).